test/e2e/tsdx-build-default.test.ts

```diff
@@ -105,6 +105,15 @@ describe('tsdx build :: zero-config defaults', () => {
     shell.mv('package-og.json', 'package.json');
   });
 
+  it('should replace lodash/fp correctly', () => {
```

Comment:
```suggestion
  it("shouldn't replace lodash/fp", () => {
```

"should replace [...] correctly" makes it sound like it's supposed to perform a replace. But it's _not_ supposed to perform a replace and the check is to make sure it's still intact